### PR TITLE
Add docker build and change layout

### PR DESCRIPTION
**Pull request by platy**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Log in to registry
+      run: docker login rg.nl-ams.scw.cloud/njkonl -u SCWQQ8SZH40EH0PCAVK9 -p ${{ secrets.SCR_PASSWORD }}
```
Comment (Focus) [inline .github/workflows/docker-image.yml]: I don't have this secret on here :)

**Review by Focus — COMMENTED** [review]
Looks much nicer than the original 👍 . Could you maybe remove the workflow and and I will push to heroku?

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (platy) [reply .github/workflows/docker-image.yml]: Sure - I've removed it

**Comment by platy** [comment]
It looks like it just took a really long time to start up, it is running now - https://covid19-per-capita.njk.onl/

**Pull request merged**